hat/assets/js/apps/Iaso/domains/users/hooks/useGetCurrentUser.ts

```diff
@@ -0,0 +1,36 @@
+import { UseQueryResult } from 'react-query';
+import { useDispatch } from 'react-redux';
+import { useSnackQuery } from '../../../libs/apiHooks';
+import { getRequest } from '../../../libs/Api';
+import { Profile } from '../../teams/types/profile';
+import { setCurrentUser } from '../actions';
+
+const getCurrentUser = (): Promise<Profile> => {
+    return getRequest('/api/profiles/me/');
+};
+export const useGetCurrentUser = (
```

Comment:
Re name change: it doesn't only get the user, it sets it in redux, so it's a bit misleading